### DISCUSSION TITLE
"Mark Tab as Unread..." as a global setting

### DISCUSSION THREAD

**Discussion by x9136** [Chatterino/chatterino2#5417] [Ideas]
![Screenshot 2024-05-24 at 12 56 38](https://github.com/Chatterino/chatterino2/assets/60105060/01014c45-ba10-4914-a734-ce377ba08d20)
I may have missed something, but it seems that it is impossible to make tabs not marked unread by default when messages appear. Now this setting is enabled for every new tab until you disable it.

I suggest adding to the Chatterino settings the ability to choose which parameter the tabs will be created with, so that it is easier to make Chatterino less distracting for those who have it open almost always. For example: **[ ] Mark unread messages for new tabs**.

**Comment by Nerixyz** [comment]
I don't really see the use-case here, but maybe I'm not familiar with your workflow. Usually people create their tabs once and do occasional adjustments. In this case, you'd disable the option _once_ for spammy channels.

If you really want to disable it for all tabs, you can overwrite the [`newMessage` tab colors](https://github.com/Chatterino/chatterino2/blob/ec6b1ef24d517c9fe6f38c910b8e36c21ec38b4c/resources/themes/Dark.json#L69-L81) in a custom theme.
